docs/troubleshooting/troubleshooting.md

```diff
@@ -38,4 +38,13 @@ The cause of the problem was that the firewall did not close the already existin
 tcpkill -9  -i ens192 src host ${KARMADA_APISERVER_IP} and dst port ${MEMBER_CLUTER_APISERVER_IP}
 ```
 
+## Error x509 is reported when `kubectl karmada init` is used to install Karmada
 
+When I use the `kubectl karmada init` command to install Karmada, the log of the `karmada-apiserver` component contains error information as follows:
```

Comment:
When using ... not `I`.

`contains`, `karmada-apiserver` doesn't `contain` the logs, it raise the log...